### PR TITLE
Update settings window to be more user friendly

### LINKED ISSUE #417: Add hide mode toggle-only
Add ability to make hide mode toggle-only, disabling the timer.

### PR DESCRIPTION
**Pull request by smkiewel**
Add helper text; rearrange, label panels; settings opens on general pane except on new version; closes #417

### CODE REVIEW

**Comment by smkiewel** [comment]
Merged new master into this branch. Did final touch-ups. Ready for final review/merging.

File: electron/settings.html [hunk 1]
```diff
@@ -51,13 +47,328 @@ <h5 class="nav-group-title">Account Settings</h5>
...
+            <input rv-checked="useMinimal" id="enable-minimal-theme-toggle" type="checkbox" data-toggle="toggle"> Enable minimal theme <span class="helper-text">Streamlines and shrinks the tracker</span>
+          </div>
+          <div class="settings-row">
+            <input rv-checked="showUIButtons" key="showUIButtons" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the close and minimize buttons
+          </div>
+          <div class="settings-row">
+                <input rv-checked="showHideButton" key="showHideButton" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the hide tracker button
+          </div>
+          <div class="settings-row">
+                <input rv-checked="showMenu" key="showMenu" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the menu button
```
Comment (shawkinsl) [inline electron/settings.html]: This toggle should be removed. Consider: user turns on this setting, closes the settings window, then wants to change some other settings. How do they get back?

**Comment by shawkinsl** [comment]
Otherwise, this PR looks great, I like how the new settings finally looks sorta professional, thanks :P 

I'm going to go ahead and merge this as is and address my one comment later.

File: electron/settings.html (hunk 1 above)
Comment (shawkinsl) [reply electron/settings.html]: oh, right! forgot about that.

OK, well, my fix will be an additional warning text here then, reminding you that you'll only be able to get to settings from the tray.

**Pull request merged**